src/core/types/Discussion.ts

```diff
@@ -0,0 +1,20 @@
+import { Optional, Address } from './Common';
+
+export type Comment = {
+  id: string;
+  replyToId: Optional<string>;
+  proposalSlug: string;
```

Comment:
is `proposalSlug` same thing as `slug`? should we unify them to `proposalId`?

Reply:
`slug` -> `proposalId`
`proposalId` -> `proposalNumber`

Thoughts?

Reply:
https://linear.app/decent-labs/issue/ENG-755/proposal-id-name-confusion